<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>583</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Map Creator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsc/Resource.qrc">
    <normaloff>:/Icons/Icons/org_unit.icns</normaloff>:/Icons/Icons/org_unit.icns</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow {background-image: url(:/Images/Images/pure_square_background.jpg);}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="NiS::BasicViewer" name="BasicViewer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="Frame_ControlPanel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Panel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="title">
                 <string>Point Cloud Density</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>20</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Max</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="HorizontalSlider_PointCloudDensity">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>10</number>
                   </property>
                   <property name="value">
                    <number>5</number>
                   </property>
                   <property name="tracking">
                    <bool>true</bool>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBelow</enum>
                   </property>
                   <property name="tickInterval">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Min</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Render Option</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QCheckBox" name="CheckBox_SpinModel">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Spin Model</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="CheckBox_ShowGrid">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show Grid</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="CheckBox_ShowPointCloud">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show Point Cloud</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="CheckBox_ShowTrajectory">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show Trajectory</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="CheckBox_ShowAnswer">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show Answer</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="CheckBox_ViewDirection_Top">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Top View</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="Line" name="line">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="PushButton_ResetModel">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Reset Model</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="PushButton_ResetCamera">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Reset Camera</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QSpinBox" name="SpinBox_EndFrame">
        <property name="maximum">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Begin Frame</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="SpinBox_BeginFrame">
        <property name="maximum">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="HorizontalSlider_BeginFrame">
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>End Frame</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="HorizontalSlider_EndFrame">
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="PushButton_Play">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="PushButton_Stop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenDataFiles"/>
    <addaction name="separator"/>
    <addaction name="actionInternalCalibration"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionOpenInliersViewMode"/>
    <addaction name="separator"/>
    <addaction name="actionShowControlPanel"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionConfigureSlamComputation"/>
    <addaction name="actionStartSlamComputation"/>
    <addaction name="separator"/>
    <addaction name="actionCaptureModelMage"/>
    <addaction name="separator"/>
    <addaction name="actionUsePreviousResult"/>
    <addaction name="actionOutputResult"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenDataFiles"/>
   <addaction name="separator"/>
   <addaction name="actionInternalCalibration"/>
   <addaction name="separator"/>
   <addaction name="actionGenerateAnswer"/>
   <addaction name="actionConfigureSlamComputation"/>
   <addaction name="actionStartSlamComputation"/>
   <addaction name="actionStopSlamComputation"/>
   <addaction name="actionCaptureModelMage"/>
   <addaction name="separator"/>
   <addaction name="actionOpenInliersViewMode"/>
   <addaction name="separator"/>
   <addaction name="actionUsePreviousResult"/>
   <addaction name="actionOutputResult"/>
   <addaction name="separator"/>
   <addaction name="actionShowControlPanel"/>
   <addaction name="actionOpenLogPanel"/>
  </widget>
  <action name="actionOpenDataFiles">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/opened_folder.icns</normaloff>:/Icons/Icons/opened_folder.icns</iconset>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
   <property name="toolTip">
    <string>Open data files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConfigureSlamComputation">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/engineering.icns</normaloff>:/Icons/Icons/engineering.icns</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure new SLAM computation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionStartSlamComputation">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/start.icns</normaloff>:/Icons/Icons/start.icns</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start new SLAM computation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionShowControlPanel">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/data_configuration.icns</normaloff>:/Icons/Icons/data_configuration.icns</iconset>
   </property>
   <property name="text">
    <string>Show Control Panel</string>
   </property>
   <property name="toolTip">
    <string>Show control panel</string>
   </property>
  </action>
  <action name="actionInternalCalibration">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/close_up_mode.icns</normaloff>:/Icons/Icons/close_up_mode.icns</iconset>
   </property>
   <property name="text">
    <string>Do / Undo internal calibration</string>
   </property>
   <property name="toolTip">
    <string>Calibrate depth image</string>
   </property>
  </action>
  <action name="actionOpenInliersViewMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/share.icns</normaloff>:/Icons/Icons/share.icns</iconset>
   </property>
   <property name="text">
    <string>Inliers View</string>
   </property>
   <property name="toolTip">
    <string>Switch to inliers viewer</string>
   </property>
  </action>
  <action name="actionCaptureModelMage">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/compact_camera.icns</normaloff>:/Icons/Icons/compact_camera.icns</iconset>
   </property>
   <property name="text">
    <string>Capture Mode lMage</string>
   </property>
   <property name="toolTip">
    <string>Capture image of current scene</string>
   </property>
  </action>
  <action name="actionUsePreviousResult">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/stack_of_photos.icns</normaloff>:/Icons/Icons/stack_of_photos.icns</iconset>
   </property>
   <property name="text">
    <string>Use Previous Result</string>
   </property>
   <property name="toolTip">
    <string>Use Previous Result</string>
   </property>
  </action>
  <action name="actionStopSlamComputation">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/disclaimer.icns</normaloff>:/Icons/Icons/disclaimer.icns</iconset>
   </property>
   <property name="text">
    <string>Stop Slam Computation</string>
   </property>
   <property name="toolTip">
    <string>Stop slam computation</string>
   </property>
  </action>
  <action name="actionIShowMarkerViewer">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/picture.icns</normaloff>:/Icons/Icons/picture.icns</iconset>
   </property>
   <property name="text">
    <string>Show Marker Viewer</string>
   </property>
   <property name="toolTip">
    <string>Show Marker Viewer</string>
   </property>
  </action>
  <action name="actionOutputResult">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/download.icns</normaloff>:/Icons/Icons/download.icns</iconset>
   </property>
   <property name="text">
    <string>Output Result</string>
   </property>
   <property name="toolTip">
    <string>Output Result</string>
   </property>
  </action>
  <action name="actionOpenLogPanel">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/survey.icns</normaloff>:/Icons/Icons/survey.icns</iconset>
   </property>
   <property name="text">
    <string>Open Log Panel</string>
   </property>
   <property name="toolTip">
    <string>Open log panel</string>
   </property>
  </action>
  <action name="actionGenerateAnswer">
   <property name="icon">
    <iconset resource="../rsc/Resource.qrc">
     <normaloff>:/Icons/Icons/mind_map.icns</normaloff>:/Icons/Icons/mind_map.icns</iconset>
   </property>
   <property name="text">
    <string>Generate answer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NiS::BasicViewer</class>
   <extends>QOpenGLWidget</extends>
   <header>../../../source/include/BasicViewer/BasicViewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rsc/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
